src/plugins/ui-router-translations/ui-router-translations-umd.js

```diff
@@ -0,0 +1,54 @@
+const { createFilter } = require('rollup-pluginutils');
+const _ = require('lodash');
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const path = require('path');
+
+const translationOutput = {};
+const translationTools = require('@ovh-ux/translations-build-tools');
+const utils = require('./utils');
+const config = require('../../config');
+
+const stateHander = (source, options) => {
+  const { translations, errors } = utils.listTranslations(source, process.cwd(), options);
+  const relativePath = path.relative(process.cwd(), options.translationsRoot);
+
+  translations.forEach((t) => {
+    // export xml translations to js
+    glob.sync(path.resolve(relativePath, t, '*.xml')).forEach((translationPath) => {
+      const xmldata = fs.readFileSync(translationPath);
+      const name = path.basename(translationPath).replace(/\.xml$/, '.json');
+      translationOutput[name] = translationOutput[name] || {};
+      translationOutput[name] = _.merge(
+        translationOutput[name],
+        JSON.parse(translationTools.translations(xmldata)),
+      );
+    });
+  });
+
+  return { result: source, errors };
+};
+
+module.exports = (opts = {}) => {
+  const options = opts;
+  if (!options.include) {
+    options.include = '**/*.js';
+  }
+  const filter = createFilter(options.include, options.exclude);
+  return {
+    name: 'ui-router-translations-umd',
+    transform(code, id) {
+      if (filter(id)) {
+        utils.uiRouterTranslations(code, stateHander, options, this.error);
+        mkdirp.sync(path.join(process.cwd(), config.dist.path.umd, 'translations'));
```

Comment:
You should resolve `process.cwd(), config.dist.path.umd, 'translations'` one time, no ? :)

if `translations` become `translations2`, we update only one var :)